spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -46,5 +52,24 @@ def test_project_explorer(project_explorer):
     assert project_explorer
 
 
+def test_copy_paste_files_or_paths(qtbot, tmpdir):
+    tmpdir.join('script.py').ensure()
+    window = DirView()
+    window.show()
+    qtbot.addWidget(window)
+    fnames = [osp.join(tmpdir, 'script.py')]
+    window.copy_path(fnames=fnames, method='absolute')
+    cb = QApplication.clipboard()
+    #  test copy absolute path
+    asb_path = cb.text(mode=cb.Clipboard)
+    cb.clear(mode=cb.Clipboard)
+    assert osp.join(tmpdir, 'script.py').replace(os.sep, "/") == asb_path
```

Comment:
You need to do ``str(tmpdir)`` here for Python 2.7 compat; that's why the tests are failing on Travis.

Reply:
Okay, sure; even better.